tmt/steps/provision/podman.py

```diff
@@ -67,16 +66,24 @@ def options(cls, how=None):
             click.option(
                 '-u', '--user', metavar='USER',
                 help='User to use for all container operations.')
-            ] + super().options(how)
+            ]
+        return options
+
+    @overload
+    def default(self, option: Literal['pull'], default: Any = None) -> bool: pass
```

Comment:
I wouldn't bother, this needs a generic solution for all plugins :/

Reply:
I got rid of the overloads. One test was failing due to importing of `Literal`. Not worth spending time to fix it.